sdk/security/keyvault/azsecrets/testdata/perf/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/Azure/azure-sdk-for-go/sdk/security/keyvault/azsecrets/testdata/perf
+
+go 1.18
+
+replace github.com/Azure/azure-sdk-for-go/sdk/security/keyvault/azsecrets v0.7.0 => ../..
```

Comment:
When a replace directive specifies a version, Go replaces only that version of the module. That means if someone changes the require version below or adds another, Go won't replace it with the local code. Probably that won't ever happen but it's better not to specify a version here because we always want to load the local code.

Reply:
fixed in [2ec3e5d](https://github.com/Azure/azure-sdk-for-go/pull/20424/commits/2ec3e5d0d03e0b875a7b70b81ed5c6e4a3b31098)